requirements.txt

```diff
@@ -4,5 +4,5 @@ requests>=2.24
 setuptools
 sphinx
 tomli>=1.2.3; python_version<'3.11'
-urllib3>=1.26.5,<2.0.0
+urllib3>=2.0.0
```

Comment:
Is this version of urllib3 available on all the supported platforms?

Reply:
@lsm5 ^^

Reply:
ummm, nope centos doesn't have `>=2.0.0`. So, looks like requirements.txt wasn't even used in `rpm-build` tasks in CI, otherwise they should've have failed.

Reply:
centos 8 stream is on v1.24 btw. So even older than the previous version specified.

Reply:
Btw, I'm checking from the spec file at: https://gitlab.com/redhat/centos-stream/rpms/python-urllib3/-/blob/c9s/python-urllib3.spec

Reply:
> ummm, nope centos doesn't have `>=2.0.0`. So, looks like requirements.txt wasn't even used in `rpm-build` tasks in CI, otherwise they should've failed.

Python has been challenged for some time with tools using requirements.txt, pyproject.toml or setup.cfg for dependency definition. pyproject.toml is the current / future, requirements.txt and setup.cfg the current / past.

pyproject.toml had dependency anchors removed back in February.

Reply:
Are we at a point where we can get rid of requirements.txt and setup.cfg, or do we still need it for something?

Reply:
@lsm5 requirements.txt is still needed for tooling that has not been updated yet.

Reply:
ack, as long as packit builds are successful, I'm cool with whatever.